src/compat_wrappers.h

```diff
@@ -8,6 +8,10 @@
 
 #include "b-em.h"
 
+#ifndef WIN32
+#define HAVE_STPCPY
+#endif
```

Comment:
@hoglet67 Dave, we do already have that check in autotools, though see my comment above about -Werror breaking that check - I think Thomas's point was that the #ifndef as you had it overwrote the result from the autotools check.

If you re-write it I have above that becomes "on Unix-like systems use the value from autotools, on Windows we know it isn't available".